### PR TITLE
Update Bulgarian translation

### PR DESCRIPTION
**Pull request by NikoVP**
Fix inconsistencies and add missing bits.

### CODE REVIEW

File: Translations/bg.po [hunk 1]
```diff
@@ -660,64 +642,69 @@ msgid "Character already exists. Do you want to overwrite \"{:s}\"?"
...
-#: Source/DiabloUI/selhero.cpp:537
+#: Source/DiabloUI/selhero.cpp:489
 msgid "Vitality:"
 msgstr "Жизненост:"
 
-#: Source/DiabloUI/selhero.cpp:548
+#: Source/DiabloUI/selhero.cpp:495
+#, fuzzy
+msgid "Savegame:"
+msgstr "Запис игра"
```
Comment (NikoVP) [inline Translations/bg.po:673]:
```suggestion
msgstr "Запис игра:"
```

File: Translations/bg.po [hunk 2]
```diff
@@ -7885,82 +7985,85 @@ msgstr "Добре, добре.  Не съм съвсем крава.  Обик
...
 
 #. TRANSLATORS: Quest text spoken by Unknown, Maybe Farmer
 #: Source/textdat.cpp:628
 msgid "Cloudy and cooler today.  Casting the nets of necromancy across the void landed two new subspecies of flying horror; a good day's work.  Must remember to order some more bat guano and black candles from Adria; I'm running a bit low."
-msgstr "Днес е облачно и хладно. Докато си вършех работата, хванах два нови видя на летящ ужас. Денят мина добре. Дано не забравя да заръчам още гуано и черни свещи на Адрия, защото ми остана малко."
+msgstr "Днес е облачно и хладно. Докато спусках мрежите на некромантството из бездната, хванах два нови видя на летящ ужас. Денят мина добре. Дано не забравя да заръчам още гуано и черни свещи на Адрия, защото ми остана малко."
```
Comment (NikoVP) [inline Translations/bg.po:8061]:
```suggestion
msgstr "Днес е облачно и хладно. Докато спусках мрежите на некромантството из бездната, хванах два нови видя на летящ ужас. Денят мина добре. Дано не забравя да заръчам още гуано и черни свещи на Адрия, защото ми остават малко."
```

File: Translations/bg.po [hunk 3]
```diff
@@ -4713,13 +4707,12 @@ msgid "Golem"
 msgstr "Голем"
 
 #: Source/monstdat.cpp:129
-#, fuzzy
 msgid "The Dark Lord"
-msgstr "Господар на мрака"
+msgstr "Тъмния Повелител"
```
Comment (NikoVP) [inline Translations/bg.po:4711]:
```suggestion
msgstr "Диабло"
```

File: Translations/bg.po [hunk 4]
```diff
@@ -400,256 +390,248 @@ msgstr "Изберете връзка"
...
 msgid "Select Difficulty"
 msgstr "Изберете трудност"
 
-#: Source/DiabloUI/selgame.cpp:163 Source/DiabloUI/selgame.cpp:210
-#: Source/DiabloUI/selgame.cpp:313 Source/DiabloUI/selgame.cpp:333
-#: Source/diablo.cpp:2202
+#: Source/DiabloUI/selgame.cpp:157 Source/DiabloUI/selgame.cpp:204 Source/DiabloUI/selgame.cpp:307 Source/DiabloUI/selgame.cpp:327 Source/diablo.cpp:1568
 #, fuzzy
 msgid "Normal"
 msgstr "Нормално"
```
Comment (NikoVP) [inline Translations/bg.po:451]:
```suggestion
msgstr "Нормалнa"
```

File: Translations/bg.po [hunk 5]
```diff
@@ -400,256 +390,248 @@ msgstr "Изберете връзка"
...
 
-#: Source/DiabloUI/selgame.cpp:164 Source/DiabloUI/selgame.cpp:214
-#: Source/diablo.cpp:2203
+#: Source/DiabloUI/selgame.cpp:158 Source/DiabloUI/selgame.cpp:208 Source/diablo.cpp:1569
 #, fuzzy
 msgid "Nightmare"
 msgstr "Кошмарно"
```
Comment (NikoVP) [inline Translations/bg.po:456]:
```suggestion
msgstr "Кошмарнa"
```

File: Translations/bg.po [hunk 6]
```diff
@@ -3646,1051 +3636,1055 @@ msgstr "Псалтски амулет"
...
+#: Source/items.cpp:4002
 #, no-c-format
 msgid "+200% damage vs. demons"
 msgstr "+200% щети срещу демони"
 
-#: Source/items.cpp:3578
+#: Source/items.cpp:4005
 msgid "All Resistance equals 0"
-msgstr "Цялaта устойчивост се равнява на 0"
+msgstr "Всичките устойчивости се равняват на 0"
```
Comment (NikoVP) [inline Translations/bg.po:4086]:
```suggestion
msgstr "Всички устойчивости са 0"
```

File: Translations/bg.po [hunk 7]
```diff
@@ -3646,1051 +3636,1055 @@ msgstr "Псалтски амулет"
...
 
-#: Source/items.cpp:3581
+#: Source/items.cpp:4008
 msgid "hit monster doesn't heal"
 msgstr "удареният враг не може да се лекува"
```
Comment (NikoVP) [inline Translations/bg.po:4090]:
```suggestion
msgstr "поразен враг не може да се лекува"
```

File: Translations/bg.po [hunk 8]
```diff
@@ -3646,1051 +3636,1055 @@ msgstr "Псалтски амулет"
...
-
-#: Source/items.cpp:3824 Source/stores.cpp:206
-msgid " {:d} Dex"
-msgstr ""
-
-#: Source/items.cpp:3835 Source/items.cpp:3882
+#: Source/items.cpp:4160 Source/items.cpp:4207
 msgid "damage: {:d}  Indestructible"
-msgstr ""
+msgstr "щети: {:d}  Неразрушим"
```
Comment (NikoVP) [inline Translations/bg.po:4249]:
```suggestion
msgstr "щети: {:d}  Неразрушими"
```

File: Translations/bg.po [hunk 9]
```diff
@@ -3646,1051 +3636,1055 @@ msgstr "Псалтски амулет"
...
 
 #: Source/monstdat.cpp:52
 msgid "Satyr Lord"
-msgstr ""
+msgstr "Сатир Господар"
 
 #: Source/monstdat.cpp:53 Source/monstdat.cpp:61
 msgid "Flesh Clan"
-msgstr ""
+msgstr "Плътен Клан"
```
Comment (NikoVP) [inline Translations/bg.po:4427]:
```suggestion
msgstr "Плътски Клан"
```

File: Translations/bg.po [hunk 10]
```diff
@@ -4840,846 +4833,855 @@ msgstr "Архиепископ Лазар"
...
 #: Source/monstdat.cpp:570
 msgid "Grimspike"
-msgstr ""
+msgstr "Мрачешип"
 
 #. TRANSLATORS: Unique Monster Block end
 #: Source/monstdat.cpp:572
 msgid "Doomlock"
-msgstr ""
+msgstr "Обреключ"
```
Comment (NikoVP) [inline Translations/bg.po:5181]:
```suggestion
msgstr "Гибелобречен"
```

File: Translations/bg.po [hunk 11]
```diff
@@ -7130,26 +7217,31 @@ msgid ""
...
+" \n"
+"Много мощен, много силен. Жив! Жив! \n"
+" \n"
+"Не болка и аз обещание спазва!"
 
 #. TRANSLATORS: Quest dialog spoken by Gharbad the Weak (Hostile)
 #: Source/textdat.cpp:303
 msgid "This too good for you. Very Powerful! You want - you take!"
-msgstr ""
+msgstr "Това твърде добро за теб. Много мощен! Ти искаш - ти получаваш!"
```
Comment (NikoVP) [inline Translations/bg.po:7229]:
```suggestion
msgstr "Това твърде добро за теб. Много мощен! Ти иска - ти получава!"
```

File: Translations/bg.po (hunk 1 above)
Comment (Krezzin) [reply Translations/bg.po]: I'd go with "Запис", closes being "Запис на игра" at the very least, instead. Currently, it sounds like a poor attempt by Google Translate. Your call.

File: Translations/bg.po (hunk 9 above)
Comment (Krezzin) [reply Translations/bg.po]: Drop the 2nd capital letter, else its fine.

**Review by Krezzin — APPROVED** [review]

File: Translations/bg.po [hunk 12]
```diff
@@ -847,358 +834,373 @@ msgstr ""
...
-#: Source/control.cpp:443
+#: Source/control.cpp:763
 #, fuzzy
 msgid "Spell Hotkey {:s}"
 msgstr "Клавишна комбинация за заклинания {:s}"
 
-#: Source/control.cpp:915
+#: Source/control.cpp:1153
 msgid "Player friendly"
 msgstr "Режим приятелски играч"
```
Comment (NikoVP) [inline Translations/bg.po:957]: Приятелска игра

File: Translations/bg.po [hunk 13]
```diff
@@ -847,358 +834,373 @@ msgstr ""
...
 
-#: Source/control.cpp:917
+#: Source/control.cpp:1155
 msgid "Player attack"
 msgstr "Режим атакуващ играч"
```
Comment (NikoVP) [inline Translations/bg.po:961]:
```suggestion
msgstr "Враждебна игра"
```

**Review by Krezzin — APPROVED** [review]
The Book of Tristram once again called upon us. Make your way through the pages, said Cain, and  disappeared in the mist.

**Comment by NikoVP** [comment]
@AJenbo it's ready for a commit. I know you have a way to fetch it from my repository, so I leave this in your capable hands. Thanks again to @Krezzin

**Comment by AJenbo** [comment]
> @AJenbo it's ready for a commit. I know you have a way to fetch it from my repository, so I leave this in your capable hands. Thanks again to @Krezzin

Ok, there is a conflict with previous edits to the file (remember to pull in the latest version before starting you work on it).
If you don't know how to resolve the conflict I can just overwrite the existing translation with what you have here, but as I don't speak Bulgarian i may end up removing something that i shoudn't :)

**Comment by NikoVP** [comment]
Yeah, you can overwrite the existing translation as this is based on the last version that was uploaded.

**Comment by AJenbo** [comment]
Merged via https://github.com/diasurgical/devilutionX/commit/69c1e2f34ad500422853ef5fbfca0b9610ed0830

You might want to reset your branch to match the upstream since there are some changes to the source text :)
There are also about 200 lines that are still marked as fuzzy and needs to be translated or validated by removing the fuzzy markation.

**Pull request closed without merging**